src/utils.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Template tag for removing whitespace and new lines
+ * in order to be able to use multiline template strings
+ * as a single string.
+ *
+ * Usage: multiConcat`foo bar baz
+ *                    whatever`;
+ *
+ * Will output: 'foo bar baz whatever'
+ *
+ */
+
+export function multiConcat(strings, ...vars) {
```

Comment:
This is all lovely, but `multiConcat` feels too vague a name. `stringConcat` feels better but the real aim is to make a single-line string, so `singleLineString` feels the best name for this function, even if it's a bit more verbose. If we import it directly:

```js
import { singleLineString } from 'utils';

var output = singleLineString`foo
               bar
         baz`;
```
looks pretty good to me.